Tests/Core/AWSClientRuntimeTests/Auth/CredentialsProvidersTests/ProcessCredentialsProviderTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ClientRuntime
+import Foundation
+import XCTest
+
+@_spi(FileBasedConfig) @testable import AWSClientRuntime
+
+class ProcessCredentialsProviderTests: XCTestCase {
+        let configPath = Bundle.module.path(forResource: "config", ofType: nil)!
+        let credentialsPath = Bundle.module.path(forResource: "credentials", ofType: nil)!
+
+    func testGetCredentialsWithDefaultProfile() async {
+        let subject = try! ProcessCredentialsProvider(
+            configFilePath: configPath,
+            credentialsFilePath: credentialsPath
+        )
+        let credentials = try! await subject.getCredentials()
+
+        XCTAssertNotNil(credentials)
```

Comment:
swift nit: This `XCTAssertNotNil` check is not needed because: 
- `let credentials = try! await subject.getCredentials()` will crash if the assignment result is `nil`
- the checks below will necessarily fail if `credentials` is nil